LibreNMS/Data/Store/InfluxDBv2.php

```diff
@@ -62,6 +63,17 @@ public static function isEnabled()
      */
     public function put($device, $measurement, $tags, $fields)
     {
+        $device_data = Device::find($device['device_id']);
```

Comment:
Device:find() is an SQL query, the device is likely already fetched at this point.  Call DeviceCache::get() instead.